app/assets/javascripts/timelines_select_boxes.js

```diff
@@ -147,24 +147,22 @@ jQuery(document).ready(function($) {
     });
   });
 
-  $("#content").find("input").each(function (index, e) {
-    e = $(e);
-    if (
-        ((e.attr("type") === "text" || e.attr("type") === "hidden") && e.val() !== "" && !e.hasClass("select2-input")) ||
-        (e.attr("type") === "checkbox" && e.attr("checked"))
-    ) {
-      showFieldSet(e);
-    }
-  });
+  var fields = $("#content").find("input")
+                            .not("[type='radio']")
+                            .not("[class^='select2-']")
+                            .not(".button")
+                            .not("[type='hidden']")
```

Comment:
@floriank I configured JSHint with `asi: false`, but I'm not 100% sure this is the right choice… if you prefer to rely on ASI then let me know / @0xF013